.env.batistack

```
@@ -0,0 +1,68 @@
+
+BCRYPT_ROUNDS=12
+
+LOG_CHANNEL=stack
+LOG_STACK=daily
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=sqlite
+DB_HOST=127.0.0.1
+DB_PORT=3306
+DB_DATABASE=laravel
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DB_DATABASE key should go before the DB_HOST key

(UnorderedKey)